### PR TITLE
Adding an impressum and changing the order of the buttons in the main…

### PR DESCRIPTION
**Pull request by jvollhueter**
In this branch an imprint was added to the homepage and the order of the main menu buttons (menu bar at the top) was changed.

### CODE REVIEW

**Comment by jbathmann** [comment]
@zaphod-bx what do you think about the impressum? should we add our e-mail contacts as well or is the connection to this github-page sufficient?
Preview: https://deploy-preview-45--pymanga.netlify.app/impressum/

**Comment by zaphod-bx** [comment]
sieht gut aus. Sollten wir unsere 4 Namen nicht drinhaben? Und das Ufz-symbol ist ja auch unten. Da könnte man auch das Ufz mit reinpappen, oder? Ein Link zur Institutsseite wäre natürlich nicht schlecht. Dann findet man auch unsere Adressen.

**Comment by jvollhueter** [comment]
In the second commit of the branch I added a contact page and adapted the main color to the corporate design of the TU Dresden. 
On the contact page the team members are introduced with a picture and their contact details (including email address, but without phone number). At the end there is an imprint for legal reasons.

What do you say to the design of the contact page? Should information be added or personal data be left out?

File: web/content/en/Impressum/_index.html [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+title: Contact
+linkTitle: Contact
+menu:
+  main:
+    weight: 4
+
+---
+
+
+{{< blocks/cover title="Das pyMANGA-Team" height="slim" color="primary">}}
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:12]: Language -> Please change to english

File: web/content/en/Impressum/_index.html [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+ <tr>
+  <td width="18%">
+    <figure>
+    <img src="Bathmann.jpeg" height="200px" width="auto">
+    <figcaption><font size = "1">&copy; Uta Berger</font></figcaption>
+    </figure><br>
+  </td>
+  <td width="47%">
+  Technische Universität Dresden<br>
+  <a href="https://tu-dresden.de/bu/umwelt/forst/ww/bsa">Professur für Forstliche Biometrie und Forstliche Systemanalyse</a><br>
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:36]: -> English

File: web/content/en/Impressum/_index.html [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+ </tr>
+ <tr>
+  <td width="18%">
+    <figure>
+    <img src="Bathmann.jpeg" height="200px" width="auto">
+    <figcaption><font size = "1">&copy; Uta Berger</font></figcaption>
+    </figure><br>
+  </td>
+  <td width="47%">
+  Technische Universität Dresden<br>
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:35]: -> English

File: web/content/en/Impressum/_index.html [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+  <a href="https://tu-dresden.de/bu/umwelt/forst/ww/bsa">Professur für Forstliche Biometrie und Forstliche Systemanalyse</a><br>
+  Pienner Straße 8<br>
+  01737 Tharandt<br>
+  <a href= "mailto:jasper.bathmann@tu-dresden.de">jasper.bathmann@tu-dresden.de</a><br>
+  &nbsp;<br>
+  &nbsp;<br>
+  </td>
+  <td width="35%">
+  Helmholtz Zentrum für Umweltforschung<br>
+  <a href="https://www.ufz.de/index.php?de=34216">Department Umweltinformatik</a><br>
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:45]: -> English

File: web/content/en/Impressum/_index.html [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+ <tr>
+  <td width="18%">
+    <figure>
+    <img src="Peters.jpeg" height="200px" width="auto">
+    <figcaption><font size = "1">&copy; Peter Frohberg</font></figcaption>
+    </figure><br>
+  </td>
+  <td width="82%">
+  Technische Universität Dresden<br>
+  <a href="https://tu-dresden.de/bu/umwelt/forst/ww/bsa">Professur für Forstliche Biometrie und Forstliche Systemanalyse</a><br>
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:74]: -> English

File: web/content/en/Impressum/_index.html [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+ <tr>
+  <td width="18%">
+    <figure>
+    <img src="Walther.jpg" height="200px" width="auto">
+    <figcaption><font size = "1">&copy; Marc Walther</font></figcaption>
+    </figure><br>
+  </td>
+  <td width="47%">
+  Technische Universität Dresden<br>
+  <a href="https://tu-dresden.de/bu/umwelt/hydro/igw">Professur für Schadstoffhydrologie</a><br>
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:103]: -> English

File: web/content/en/Impressum/_index.html [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+ <tr>
+  <td width="18%">
+    <figure>
+    <img src="Berger.jpg" height="200px" width="auto">
+    <figcaption><font size = "1">&copy; Harald Schluttig</font></figcaption>
+    </figure><br>
+  </td>
+  <td width="82%">
+  Technische Universität Dresden<br>
+  <a href="https://tu-dresden.de/bu/umwelt/forst/ww/bsa">Professur für Forstliche Biometrie und Forstliche Systemanalyse</a><br>
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:136]: -> English

File: web/content/en/Impressum/_index.html [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+ <tr>
+  <td width="40%" bgcolor="#00305E" id="kopf" colspan="1" ><h4> <font color = "white"> Co-developer </font> </h4></td>
+  <td width="10%"> &nbsp; </td>
+  <td width="40%" bgcolor="#00305E" id="kopf" colspan="1"><h4> <font color = "white"> Sponsor </font> </h4></td>
+ <tr>
+  <td width="40%">
+  Tom Franitza<br>
+  Wissenschaftliche Hilfskraft<br>
+  Technische Universität Dresden<br>
+  <hr style="height: 2px; background-color: black; border: 0;" />
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:163]: -> English

File: web/content/en/Impressum/_index.html [hunk 9]
```diff
@@ -0,0 +1,215 @@
...
+  Wissenschaftliche Hilfskraft<br>
+  Technische Universität Dresden<br>
+  <hr style="height: 2px; background-color: black; border: 0;" /> 
+  Jonas Vollhüter<br>
+  Wissenschaftliche Hilfskraft<br>
+  Technische Universität Dresden<br>
+  <hr style="height: 2px; background-color: black; border: 0;" /> 
+  Chris Wudel<br>
+  Wissenschaftliche Hilfskraft<br>
+  Technische Universität Dresden<br>
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:170]: -> English

File: web/content/en/Impressum/_index.html [hunk 10]
```diff
@@ -0,0 +1,215 @@
...
+  <td width="40%" bgcolor="#00305E" id="kopf" colspan="1"><h4> <font color = "white"> Sponsor </font> </h4></td>
+ <tr>
+  <td width="40%">
+  Tom Franitza<br>
+  Wissenschaftliche Hilfskraft<br>
+  Technische Universität Dresden<br>
+  <hr style="height: 2px; background-color: black; border: 0;" /> 
+  Jonas Vollhüter<br>
+  Wissenschaftliche Hilfskraft<br>
+  Technische Universität Dresden<br>
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:166]: -> English

File: web/content/en/Impressum/_index.html [hunk 11]
```diff
@@ -0,0 +1,215 @@
...
+{{<blocks/section color="primary" heigt="slim">}}
+
+<table border="0" width="100%" cellspacing="0" cellpadding="5" summary="Layout" style="text-align: center">
+ <tr>
+  <td width="100%">
+  <font size = "7">Impressum:</font><br>
+   &nbsp; <br>
+   Prof. Dr. rer. nat. Uta Berger <br>
+   Technische Universität Dresden <br>
+   Professur für Forstliche Biometrie und Forstliche Systemanalyse<br>
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:195]: -> English

**Comment by jbathmann** [comment]
Great! Just the language stuff before we are able to merge. :-)

**Comment by jvollhueter** [comment]
In the latest commit the language problems on the english contact page are fixed. I wasn't quite sure if "Technische Universität Dresden" should be translated into english, because I read in a notice from the university that the german name should be used in english as well. Should the name be translated or not?

**Comment by jbathmann** [comment]
> In the latest commit the language problems on the english contact page are fixed. I wasn't quite sure if "Technische Universität Dresden" should be translated into english, because I read in a notice from the university that the german name should be used in english as well. Should the name be translated or not?

I think, we should follow the University reugulations and stick to "Technische Universität". Thanks for the updates!

File: web/content/en/Impressum/_index.html [hunk 12]
```diff
@@ -0,0 +1,216 @@
...
+  Graduate Assistant<br>
+  Technical University of Dresden<br>
+  <hr style="height: 2px; background-color: black; border: 0;" /> 
+  Jonas Vollhüter<br>
+  Graduate Assistant<br>
+  Technical University of Dresden<br>
+  <hr style="height: 2px; background-color: black; border: 0;" /> 
+  Chris Wudel<br>
+  Graduate Assistant<br>
+  Technical University of Dresden<br>
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:170]: Either "Dresden University of Technology" or (better) the german version

File: web/content/de/Impressum/_index.html [hunk 13]
```diff
@@ -0,0 +1,213 @@
...
+
+
+
+<table border="0" width="100%" cellspacing="0" cellpadding="5" summary="Layout">
+ <tr>
+  <td colspan="2" bgcolor="#00305E" id="kopf" ><h4> <font color = "white"> Prof. Dr. rer. nat. Uta Berger </font> </h4></td>
+ <tr>
+  <td width="18%">
+    <figure>
+    <img src="Berger.jpeg" height="200px" width="auto">
```
Comment (jbathmann) [inline web/content/de/Impressum/_index.html:128]: This file does not exist. --> 
Berger.jpg

Comment (jbathmann) [inline web/content/de/Impressum/_index.html:128]:
```suggestion
    <img src="Berger.jpg" height="200px" width="auto">
```

File: web/content/en/Impressum/_index.html [hunk 14]
```diff
@@ -0,0 +1,216 @@
...
+ </tr>
+ <tr>
+  <td width="18%">
+    <figure>
+    <img src="Bathmann.jpeg" height="200px" width="auto">
+    <figcaption><font size = "1">&copy; Uta Berger</font></figcaption>
+    </figure><br>
+  </td>
+  <td width="47%">
+  Technical University of Dresden<br>
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:35]:
```suggestion
  Technische Universität Dresden<br>
```

File: web/content/en/Impressum/_index.html [hunk 15]
```diff
@@ -0,0 +1,216 @@
...
+ </tr>
+ <tr>
+  <td width="18%">
+    <figure>
+    <img src="Peters.jpeg" height="200px" width="auto">
+    <figcaption><font size = "1">&copy; Peter Frohberg</font></figcaption>
+    </figure><br>
+  </td>
+  <td width="82%">
+  Technical University of Dresden<br>
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:73]:
```suggestion
  Technische Universität Dresden<br>
```

File: web/content/en/Impressum/_index.html [hunk 16]
```diff
@@ -0,0 +1,216 @@
...
+ </tr>
+ <tr>
+  <td width="18%">
+    <figure>
+    <img src="Walther.jpg" height="200px" width="auto">
+    <figcaption><font size = "1">&copy; Marc Walther</font></figcaption>
+    </figure><br>
+  </td>
+  <td width="47%">
+  Technical University of Dresden<br>
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:102]:
```suggestion
  Technische Universität Dresden<br>
```

File: web/content/en/Impressum/_index.html [hunk 17]
```diff
@@ -0,0 +1,216 @@
...
+  <td colspan="2" bgcolor="#00305E" id="kopf" ><h4> <font color = "white"> Prof. Dr. rer. nat. Uta Berger </font> </h4></td>
+ <tr>
+  <td width="18%">
+    <figure>
+    <img src="Berger.jpg" height="200px" width="auto">
+    <figcaption><font size = "1">&copy; Harald Schluttig</font></figcaption>
+    </figure><br>
+  </td>
+  <td width="82%">
+  Technical University of Dresden<br>
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:135]:
```suggestion
  Technische Universität Dresden<br>
```

File: web/content/en/Impressum/_index.html [hunk 18]
```diff
@@ -0,0 +1,216 @@
...
+<table border="0" width="100%" cellspacing="0" cellpadding="5" summary="Layout">
+ <tr>
+  <td width="40%" bgcolor="#00305E" id="kopf" colspan="1" ><h4> <font color = "white"> Co-developer </font> </h4></td>
+  <td width="10%"> &nbsp; </td>
+  <td width="40%" bgcolor="#00305E" id="kopf" colspan="1"><h4> <font color = "white"> Sponsor </font> </h4></td>
+ <tr>
+  <td width="40%">
+  Tom Franitza<br>
+  Graduate Assistant<br>
+  Technical University of Dresden<br>
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:162]:
```suggestion
  Technische Universität Dresden<br>
```

File: web/content/en/Impressum/_index.html [hunk 19]
```diff
@@ -0,0 +1,216 @@
...
+  <hr style="height: 2px; background-color: black; border: 0;" /> 
+  Jonas Vollhüter<br>
+  Graduate Assistant<br>
+  Technical University of Dresden<br>
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:166]:
```suggestion
  Technische Universität Dresden<br>
```

File: web/content/en/Impressum/_index.html (hunk 12 above)
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:170]:
```suggestion
  Technische Universität Dresden<br>
```

File: web/content/en/Impressum/_index.html [hunk 20]
```diff
@@ -0,0 +1,216 @@
...
+
+{{<blocks/section color="primary" heigt="slim">}}
+
+<table border="0" width="100%" cellspacing="0" cellpadding="5" summary="Layout" style="text-align: center">
+ <tr>
+  <td width="100%">
+  <font size = "7">Imprint:</font><br>
+   &nbsp; <br>
+   Prof. Dr. rer. nat. Uta Berger <br>
+   Technical University of Dresden <br>
```
Comment (jbathmann) [inline web/content/en/Impressum/_index.html:194]:
```suggestion
  Technische Universität Dresden <br>
```

**Pull request merged**